.idea/gradle.xml

```diff
@@ -20,7 +20,8 @@
           </set>
         </option>
         <option name="resolveModulePerSourceSet" value="false" />
+        <option name="useQualifiedModuleNames" value="true" />
       </GradleProjectSettings>
     </option>
   </component>
-</project>
+</project>
```

Comment:
Undo these changes.

Reply:
Done

Reply:
Doesn't seem done--did you push the changes?

Reply:
Yes I have pushed changes. It is autogenerating. How can I fix this?

Reply:
Yes, its an autogenerating thing, I have been ignoring this while adding files to commit.

Reply:
You can remove it by using ``git checkout .idea/gradle.xml`` locally, committing that, then keeping an eye out for .idea changes when committing changes.

Reply:
Fixed this.